theseus/embodied/motionmodel/double_integrator.py

```diff
@@ -118,6 +118,9 @@ def __init__(
         )
         self.register_aux_vars(["Qc_inv", "dt"])
 
+    def is_zero(self) -> torch.Tensor:
+        raise NotImplementedError
```

Comment:
Should this return a more reasonable value? We know GPCostWeight is non zero. Or we could make the `is_zero` function not abstract so every derived class doesn't need to define it?

Reply:
Changed as discussed offline. I'd prefer to make abstract, because this is always used under the hood, and there is no obvious default.